plugins/transport-grpc.yml

```diff
@@ -0,0 +1,11 @@
+id: transport-grpc
+name: gRPC API
+description: This plugin adds gRPC endpoints to your generated service
+repo: amplication/plugins
+npm: '@amplication/plugin-transport-grpc'
```

Comment:
do we want to rename the plugin to `@amplication/plugin-api-grpc` for consistency?

Reply:
@overbit I considered but I think it's too much work for the value we will get out of this rename. It's already published to NPM, it's changing several things in the plugin itself + re-publish to NPM as a new package + update this PR - and the value from all of this is not really that meaningful IMO